opcua/ua/ua_binary.py

```diff
@@ -503,6 +503,9 @@ def struct_from_binary(objtype, data):
             if not test_bit(val, idx):
                 continue
         val = from_binary(uatype, data)
+        if val is None and uatype[:6] == 'ListOf':
+            val = []
+            
```

Comment:
What is the issue exactly?the UA protocols allows to return Null values which are best translated to None in Python. So the original code looked correct